modules/nf-core/dshbio/filterbed/tests/main.nf.test

```diff
@@ -0,0 +1,63 @@
+
+nextflow_process {
+
+    name "Test Process DSHBIO_FILTERBED"
+    script "../main.nf"
+    process "DSHBIO_FILTERBED"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "dshbio"
+    tag "dshbio/filterbed"
+
+    test("test-dshbio-filterbed") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test' ], // meta map
+				    [ file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/bed/test.bed', checkIfExists: true) ]
+				]
+
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    file(process.out.bed[0][1]).name, // empty
```

Comment:
Why do you only test for the file name? Is the bed not stable enough to check the md5sum?

Reply:
Don't have this checked out at the moment, but yes it probably is empty.  The filter looks to be `--range chr1:0-1000` which won't match anything from sarscov2 genome.

Reply:
Can we change it to something that leads to a non-empty file which can then be checked with md5sums?

Reply:
Thank you @heuermh and @famosab 

I have updated the params so that the output is not empty.